fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/PhysicalPlanTranslator.java

```diff
@@ -125,10 +126,12 @@ public PlanFragment translatePlan(PhysicalPlan physicalPlan, PlanTranslatorConte
         // TODO: trick here, we need push project down
         if (physicalPlan.getType() == PlanType.PHYSICAL_PROJECT) {
             PhysicalProject<Plan> physicalProject = (PhysicalProject<Plan>) physicalPlan;
-            List<Expr> outputExprs = physicalProject.getProjects().stream()
-                    .map(e -> ExpressionTranslator.translate(e, context))
-                    .collect(Collectors.toList());
-            rootFragment.setOutputExprs(outputExprs);
+            rootFragment.setOutputExprs(
+                    physicalProject
+                            .getOutput()
+                            .stream()
+                            .map(s -> ExpressionTranslator.translate(s, context))
+                            .collect(Collectors.toList()));
```

Comment:
why not put projection info on the root PlanNode and remove this branch

Reply:
`outputExprs` is set in the root fragment only, it's not needed by every fragment